package/lib/SimpleSchema.js

```diff
@@ -91,12 +92,22 @@ class SimpleSchema {
     this._depsLabels = {};
     this.extend(schema);
 
+    // Clone raw definition and save if keepRawDefinition is active
+    this._rawDefinition = keepRawDefinition && schema;
```

Comment:
Change this to `this._rawDefinition = keepRawDefinition ? schema : null;`. It makes more sense for it to be `null` rather than `false`.